src/standalone/mockttp-standalone.ts

```diff
@@ -128,6 +128,15 @@ export class MockttpStandalone {
         const mockServerRouter = express.Router();
         this.routers[mockPort] = mockServerRouter;
 
+        mockServerRouter.get('/pendingMocks', async (req, res) => {
+            try {
+                var checkRequests = await mockServer.pendingMocks();
+                res.status(200).send(JSON.stringify(checkRequests));
+            } catch (error) {
+                res.status(500).send(JSON.stringify({ message : error.message }));
+            }
+        });
```

Comment:
This should actually live in the GraphQL, like `mockedEndpoints`. That bit of the code might be confusing if you haven't worked with GraphQL before - let me know if you want me to help out there.

Reply:
I didnt work with GraphQl before so I would prefer if you would help me to fix code.